src/genomedata_load_data.c

```diff
@@ -117,6 +117,42 @@ typedef struct {
 #endif /* GCC_VERSION >= 20500*/
 #endif /* __GNUC__ */
 
+void load_data(char* gdfilename, char* trackname, bool verbose);
+
+static PyObject* load_data_from_stdin(PyObject* self, PyObject* args) {
+  const char* gdfilename;
+  const char* trackname = "dummy";
+  int verbose; /* needs to be int, not bool for parsing */
+
+  if (!PyArg_ParseTuple(args, "ssp", &gdfilename, &trackname, &verbose)) /* string string bool predicate */
```

Comment:
I think the `bool` is actually the `predicate`.

Reply:
It is. This comment is supposed to clarify why there is a special string called "ssp" and not "ssb" or something to that effect.

Reply:
Maybe the comment should be changed to `/* string string predicate */` then?